content/02.main-text.md

```diff
@@ -39,15 +39,19 @@ Furthermore, none of these platforms offer the ability to address thematically r
 | Document format | Markdown | LaTeX | LaTeX | Proprietary | Proprietary | Markdown |
 | Templating | Yes | Yes | Yes | No | No | No |
 | Technical expertise required | Yes | Yes | Yes | No | No | Yes |
+| WYSIWYG mode | No | Rich text available | Rich text available | Yes | Yes | Preview rendered Markdown |
+| Inline comments | Yes using Hypothesis | Yes | Yes | Yes | Yes | No |
```

Comment:
by "Inline comments" do you mean "Reader comments"? I could also interpret this as perhaps discussion between authors or reviewers and authors.

Reply:
"Inline comments" is the phrase the reviewer used.  We could change it to be more precise.  My interpretation was comments that persist with manuscript, with comments in Word or Google Docs being the canonical example.  These are primarily used for inline discussion between authors.

Do you have an idea for how to change the feature name or the entries in the row?  This was only my first attempt and could be improved.

Reply:
Makes sense. I think "inline comments" is correct then if we're referring to persistent comments made during the authoring stage.